File: Employee Directory/ClientApp/src/app/models/Employee.ts
export class Employee{
    public id:number;
    public firstName: string;
    public lastName: string;
    public preferredName: string;
    public email: string;
    public phoneNumber: string;
    public skypeId: string;
    public departmentId: number;
    public officeId: number;
    public jobTitleId: number;

    constructor(args: any){
        args = !!args ? args : {};
        this.id = args.id;
        this.firstName = args.firstName;
        this.lastName = args.lastName;
        this.preferredName = args.preferredName;
        this.email = args.Email;
        this.phoneNumber = args.phoneNumber;
        this.skypeId = args.skypeId;
        this.departmentId = args.departmentId;
        this.officeId = args.officeId;
        this.jobTitleId = args.jobTitleId;
        
    }
}

File: Models/Employee.cs
﻿namespace Employee_Directory.Models
{
    public class Employee
    {
        public int Id { get; set; }

        public string FirstName { get; set; }

        public string LastName { get; set; }

        public string PreferredName { get; set; }

        public string Email { get; set; }

        public string Department { get; set; }

        public string JobTitle { get; set; }

        public string Office { get; set; }

        public string PhoneNumber { get; set; }

        public string SkypeId { get; set; }

        public int DepartmentId { get; set; }

        public int OfficeId { get; set; }

        public int JobTitleId { get; set; }
    }
}
